lib/note-content-editor.jsx

```diff
@@ -233,10 +233,10 @@ export default class NoteContentEditor extends Component {
   };
 
   handleReturn = () => {
-    // matches lines that start with `- `, `* `, or `+ `
+    // matches lines that start with `- `, `* `, `+ `, or `•`
     // preceded by 0 or more space characters
     // i.e. a line prefixed by a list bullet
-    const listItemRe = /^[ \t\u2000-\u200a]*[-*+]\s/;
+    const listItemRe = /^[ \t\u2000-\u200a]*[-*+•]\s/;
```

Comment:
Looks like this is fine when I test in my terminal, so it's probably good to leave, but if we are worried about issues having Unicode symbols in the source we can replace this with `\u2022`, clearing up any confusion about which symbol we wanted.

Related but unlikely and unwanted additions might also be:
 - bullet operator `\u2219` `∙`
 - bullet train `\u1f685` `🚅`
 - bullet bullet `\u204c` `⁌`